libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -63,6 +76,11 @@ impl RestResourceProvider for AdminService {
 /// * `GET /admin/circuits` - List circuits in Splinter's state
 /// * `GET /admin/circuits/{circuit_id}` - Fetch a specific circuit in Splinter's state by circuit
 ///   ID
+///
+/// These endpoints are only available if one of the following REST API backend features is
+/// enabled:
+///
+/// * `rest-api-actix`
```

Comment:
Are more features expected for this list?  If not, I'd pull the bullet item into the sentence; for example:  ".... only available if the REST API back-end feature `rest-api-actix` is enabled."  Otherwise, I'd change "one of the following ... features" to "the following ... feature" (so that the text matches the number of items in the list). 

Ditto for the similar doc comments on lines 102-105.

Reply:
Yes, more features are expected for this list.

Reply:
In that case, I recommend correcting the text to match the number of items currently on the list: "the following REST API back-end feature is enabled"). 

Later, when more bullets are added, the text can change to "any of the following...."

Reply:
Updated

Reply:
Per @chenette comment, should this say `REST API backend feature` instead of `REST API backend features`? Same for the comment bellow.

Reply:
Oof. Fixed.